app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,85 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+  skip_before_action :authenticate_user!, only: %i[public_recipes show]
+
+  # GET /recipes or /recipes.json
+  def index
+    # logged in user recipes
+    @user_test = current_user
+    @recipes = Recipe.where(user: @user_test)
+  end
+
+  def public_recipes
+    @recipes = Recipe.includes(:recipe_foods, :foods).all
+  end
+
```

Comment:
- [ ] Great job so far 👏🏻 However the public recipes page should only list all `publicly` available recipes from other users with their names and total prices. Actually, it's lists all the created recipes even those which are not public as well, To fix this you only need to modify your line 13 or you can also do it from the view if you want ✍🏻 


- I can see that your Page has to match the structure of the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true)
 Great job on the design 😉 👍🏻 🥇

Reply:
@BenMukebo did you try singing out? you can see all public and none public recipes because you are logged as Ben.
The wireframe is changed as suggestedl